<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiConfigDialog</class>
 <widget class="QDialog" name="UiConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>界面配置</string>
  </property>
  <widget class="QWidget" name="w" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>411</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#w
{
border: 2px solid gray;
border-radius:5px; 
padding: 0px 10px 0px 20px;
 
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="w0" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#w01
{
border: 2px solid gray;
border-radius:5px; 
padding: 0px 10px 0px 20px;
 
}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Adobe 仿宋 Std R</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>横向数量:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Adobe 仿宋 Std R</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>纵向数量:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="font">
          <font>
           <family>幼圆</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit">
         <property name="font">
          <font>
           <family>幼圆</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>130</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>幼圆</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>客户端横纵数量</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="w1" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>290</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#w11
{
border: 2px solid gray;
border-radius:5px; 
padding: 0px 10px 0px 20px;
 
}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="text">
          <string>图片缺失</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="text">
          <string>图片缺失</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" alignment="Qt::AlignHCenter">
        <widget class="QRadioButton" name="radioButton_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>幼圆</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>纵向</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>幼圆</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>客户端摆放形式</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0" alignment="Qt::AlignHCenter">
        <widget class="QRadioButton" name="radioButton">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>幼圆</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>横向</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>教师机位置</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="w2" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#w21
{
border: 2px solid gray;
border-radius:5px; 
padding: 0px 10px 0px 20px;
 
}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>幼圆</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>更多设置</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="font">
          <font>
           <family>幼圆</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string>例如A0 A1 A2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <family>Adobe 仿宋 Std R</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>起始位置：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="font">
          <font>
           <family>幼圆</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string>例如A0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <family>Adobe 仿宋 Std R</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>缺失位置：</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2" alignment="Qt::AlignRight">
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">QLabel
{
	color:rgb(255, 0, 0)
}</string>
         </property>
         <property name="text">
          <string>输入缺失的机位号,以空格分隔</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="w3" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#w3
{
border: 1px solid gray;
border-radius:5px; 
padding: 0px 10px 0px 20px;
background:rgb(214, 214, 214)
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>radioButton</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
